test/smtcallback.js

```diff
@@ -167,9 +169,8 @@ tape('SMTCheckerCallback', function (t) {
       if (source.includes(option)) {
         let idx = source.indexOf(option);
         if (source.indexOf(option, idx + 1) !== -1) {
-          st.skip('SMTEngine option given multiple times.');
-          st.end();
-          return;
```

Comment:
This was actually making none of the actual smtcheckertests to run lol

Reply:
This is the main fix, I reckon?

Reply:
The `solverFunction` and `solver` fixes in `smtsolver.js` too, solvers were actually not run